test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -60,6 +60,57 @@ await foreach (Page<BinaryData> page in allPages.AsPages())
             }
         });
 
+        [Test]
+        public Task HandwrittenModelLro() => Test(async (host) =>
+        {
+            using var diagnosticListener = new ClientDiagnosticListener("dpg_customization_LowLevel", asyncLocal: true);
+            CollectionAssert.IsEmpty(diagnosticListener.Scopes);
+
+            var lro = await new DPGClient(Key, host).LroValueAsync(WaitUntil.Started, "model");
+            Assert.AreEqual(1, diagnosticListener.Scopes.Count);
+            Assert.AreEqual("DPGClient.LroValue", diagnosticListener.Scopes[0].Name);
+
+            await lro.WaitForCompletionAsync();
+            Assert.AreEqual(2, diagnosticListener.Scopes.Count);
```

Comment:
Why two scopes here?  Are they nested?

Reply:
No. One has been created by `LroValueAsync` and another one by `WaitForCompletionAsync`